installer/templates/phx_single/config/prod.secret.exs

```diff
@@ -13,5 +13,5 @@ secret_key_base =
     """
 
 config :<%= app_name %>, <%= endpoint_module %>,
-  http: [:inet6, port: String.to_integer(System.get_env("PORT") || "4000")],
+  http: [:inet6, ip: {127, 0, 0, 1}, port: String.to_integer(System.get_env("PORT") || "4000")],
```

Comment:
This fails because when the `:inet6` flag is given we need to pass an IPv6 address for `:ip`. From some basic testing on my local machine it looks like we can specify the IPv6 loopback address and it will work with both IPv6 and IPv4.
```suggestion
  http: [:inet6, ip: {0, 0, 0, 0, 0, 0, 0, 1}, port: String.to_integer(System.get_env("PORT") || "4000")],
```

Reply:
> we can specify the IPv6 loopback address and it will work with both IPv6 and IPv4.

Actually.. only `localhost` works. If it's bound to ipv6 loopback (`::1`) then ipv4 loopback (`127.0.0.1`) wouldn't work.

```sh
curl -v 127.0.0.1:4000
*   Trying 127.0.0.1...
* TCP_NODELAY set
* Connection failed
* connect to 127.0.0.1 port 4000 failed: Connection refused
* Failed to connect to 127.0.0.1 port 4000: Connection refused
* Closing connection 0
curl: (7) Failed to connect to 127.0.0.1 port 4000: Connection refused
```

```sh
curl -v localhost:4000
*   Trying ::1...
* TCP_NODELAY set
* Connected to localhost (::1) port 4000 (#0)
> GET / HTTP/1.1
> Host: localhost:4000
> User-Agent: curl/7.64.1
> Accept: */*
>
< HTTP/1.1 404 Not Found
(...omitted...)
```